database/seed.js

```diff
@@ -33,60 +33,63 @@ const booking_schema = new mongoose.Schema({
   weeknight_discount: Number,
   instant_book: Boolean,
   request_to_book: Boolean,
-  addons_names: [String],
-  addons_price: Number,
   subTotal: Number
 });
 
 const Booking = mongoose.model('Booking', booking_schema);
 
 const seedDb = async () => {
-
   for (let i = 0; i <= 99; i++) {
-
     monthMaker = function() {
```

Comment:
all of there make functions don't need to be inside this for-loop. Not sure if it's more effcient to move them outside so that you dont have to remake all these functions every loop. It looks like most of them are constant and don't depend on the position in the loop. 

secondy and just a side note. the faker npm package could probably have helped in getting random dates, etc. but it's cool to see you code it all out!

Reply:
Lol yea wasn't even considering performance. Moved all the make funs out of the for loop. Yea, guess I did it the hard way but had fun